packages/tenant-process/src/services/tenantService.ts

```diff
@@ -1189,6 +1190,100 @@ export function tenantServiceBuilder(
       }
       return tenant.data;
     },
+
+    async m2mRevokeCertifiedAttribute({
+      organizationId,
+      tenantOrigin,
+      tenantExternalId,
+      attributeExternalId,
+      correlationId,
+      logger,
+    }: {
+      organizationId: TenantId;
+      tenantOrigin: string;
+      tenantExternalId: string;
+      attributeExternalId: string;
+      correlationId: string;
+      logger: Logger;
+    }): Promise<void> {
+      logger.info(`Revoking certified attribute to tenant`);
```

Comment:
This is the current log
```suggestion
      logger.info(`Revoking certified attribute $”{attributeExternalId} to tenant (${tenantOrigin}/${tenantExternalId}) via m2m request`);
```

Reply:
Updated